draft/XXXX-robust-api/README.md

```diff
@@ -0,0 +1,146 @@
+---
+RFC: XXXX
+Author: Jacob Nesbitt
+Created: 2020-06-29
+Created: 2020-12-12
```

Comment:
```suggestion
```

Reply:
Good catch, not sure why that was there.